features/restaurant_page.feature

```diff
@@ -0,0 +1,20 @@
+Feature: User can see restaurant's details on the restaurant show page
+  As a user
+  In order to get info about the restaurant
+  I would like to see restaurant details
+
+  Background:
+    Given the following restaurants exist
+      | name              | description                     |
+      | Thomas Kebabrulle | The best kebabrulle you can get |
+      | Oliver's Burger   | The best burger at Chalmers     |
+      | Borat's Palace    | Jak sie masz?                   |
+
+    When I visit the landing page
```

Comment:
This is a bit of a premature refactoring IMO. You only have one scenario in this feature how do you know you'll need to perform these 3 steps from visiting the landing downward for every scenario to follow?

I'd just move them to the scenario below for now and only extract them to the background step if really necessary.